specification/azsadmin/resource-manager/subscriptions/readme.md

```diff
@@ -107,7 +107,7 @@ AutoRest V3 generators require the use of `--tag=all-api-versions` to select api
 
 This block is updated by an automatic script. Edits may be lost!
 
-``` yaml
+``` yaml $(tag) == 'all-api-versions'
 # include the azure profile definitions from the standard location
 require: $(this-folder)/../../../../profiles/readme.md
 
```

Comment:
We could remove this whole section of "Multi-API/Profile support for AutoRest v3 generators"
It is auto-generated previously and now deprecated, we do not need to maintain this section.